doc/source/reference/routines.ma.rst

```diff
@@ -247,7 +247,7 @@ Conversion operations
    ma.masked_where
 
 
-> to a ndarray
+> to an ndarray
 ~~~~~~~~~~~~~~
```

Comment:
```suggestion
~~~~~~~~~~~~~~~
```
FWIW, I guess for most of these its a small improvement. I honestly don't care what we do with `np.`, considering them to be read as "numpy..." would be just as well to me (I think I ).
But maybe we should just stick with the old one here for the sake of fewer changes?